src/features_creator/features_creator.py

```diff
@@ -38,3 +39,46 @@ def get_matching_column_names(data, pattern):
 
     """
     ...
+
+
+
+def calculate_standard_deviation(data, cols):
+    """Returns a dataframe with standard deviation of specific columns.
+
+    Calculating standard deviation of columns inputed.
+
+    Parameters
+    ----------
+    data : pandas dataframe
+        The dataframe to calculate standard deviation
+    cols : list
```

Comment:
When I was picturing this function, I thought `cols` would be a string pattern just like in `get_matching_column_names`, then this function would internally call `get_matching_column_names`. This way, the user would only need to call `calculate_standard_deviation` without also having to call `get_maching_column_names`. What do you think?

Reply:
I think this is fine, otherwise, we would have to state some range from 1-3 for example. The pattern_matching parameter can be optional, but I wouldn't press it right now

Reply:
I am fine with either way. Maybe we just need to keep a consistency, if we decide to use pattern in every function, I will revise the code and doc.